api/pom.xml

```diff
@@ -9,22 +9,13 @@
     <version>0.1.0-SNAPSHOT</version>
   </parent>
 
-  <artifactId>smallrye-opentelemetry</artifactId>
+  <artifactId>smallrye-opentelemetry-api</artifactId>
 
-  <profiles>
-    <profile>
-      <id>coverage</id>
-      <properties>
-        <argLine>@{jacocoArgLine}</argLine>
-      </properties>
-      <build>
-        <plugins>
-          <plugin>
-            <groupId>org.jacoco</groupId>
-            <artifactId>jacoco-maven-plugin</artifactId>
-          </plugin>
-        </plugins>
-      </build>
-    </profile>
-  </profiles>
+  <dependencies>
+    <dependency>
+      <groupId>javax.enterprise</groupId>
```

Comment:
This should be the jakarta artifact

Reply:
done updated all artifacts to `jakarta`.